site/en/r2/tutorials/load_data/numpy.ipynb

```diff
@@ -0,0 +1,311 @@
+        "colab": {}
+      },
+      "source": [
+        "!pip install tensorflow==2.0.0-alpha0"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "k6J3JzK5NxQ6",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "import numpy as np\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_datasets as tfds"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "G0yWiN8-cpDb",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Load from `.npz` file"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "GLHNrFM6RWoM",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "DATA_URL = 'https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz'\n",
+        "\n",
+        "path = tf.keras.utils.get_file('mnist.npz', DATA_URL)\n",
+        "with np.load(path) as data:\n",
+        "  train_examples = data['x_train']\n",
+        "  train_labels = data['y_train']\n",
+        "  test_examples = data['x_test']\n",
+        "  test_labels = data['y_test']"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cCeCkvrDgCMM",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Load Numpy arrays with `tf.data.Dataset`"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tslB0tJPgB-2",
+        "colab_type": "text"
+      },
+      "source": [
+        "Assuming you have an array of examples and a corresponding array of labels, pass the two arrays as a tuple into `tf.data.Dataset.from_tensor_slices` to create a `tf.data.Dataset`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "QN_8wwc5R7Qm",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "train_dataset = tf.data.Dataset.from_tensor_slices((train_examples, train_labels))\n",
+        "test_dataset = tf.data.Dataset.from_tensor_slices((test_examples, test_labels))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6Rco85bbkDfN",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Use the datasets"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0dvl1uUukc4K",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Prepare the datasets\n",
+        "\n",
+        "For this examples, we're going to flatten the input data, shuffle it, and batch it."
```

Comment:
Prefer "you" instead if "we"
https://developers.google.com/style/person

Reply:
fixed